regression/pages/studio/grading_studio.py

```diff
@@ -46,8 +42,47 @@ def click_remove_grade(self):
         btn_css = '.remove-button'
         self.browser.execute_script("$('{}').focus().click()".format(btn_css))
         self.wait_for_ajax()
-        press_the_notification_button(self, "save")
-        self.wait_for_element_visibility(
-            '#alert-confirmation-title',
-            'Save confirmation message is visible'
-        )
+        save_changes_popup_for_studio(self)
+
+    def click_new_assignment_type(self):
+        """
+        Clicks New Assignment type button on Grading page
+        """
+        self.q(css='.add-grading-data').click()
+        save_changes_popup_for_studio(self)
+
+    def fill_assignment_type_fields(
+            self, name, abbreviation,
+            total_grade, total_number, drop):
+        """
+        Fills text to Assignment Type fields according to assignment box
+        number and text provided
+        Args:
+            name: Assignment Type Name
+            abbreviation: Abbreviation
+            total_grade: Weight of Total Grade
+            total_number: Total Number
+            drop: Number of Droppable
```

Comment:
@raeeschachar Can you update the docstring as per [edx python guide](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings)